buildpacks/nodejs-engine/tests/integration_test.rs

```diff
@@ -0,0 +1,59 @@
+#[cfg(test)]
+use libcnb_test::{BuildpackReference, IntegrationTest};
+use std::time::Duration;
+use ureq;
+
+#[test]
```

Comment:
For our other buildpacks, we've added an ignore attribute to allow running unit tests and integration tests separately. See:
https://github.com/heroku/procfile-cnb/pull/35
https://github.com/Malax/libcnb.rs/pull/316